doc/ProductSearch.md

```diff
@@ -192,6 +192,15 @@ public void yourFunction(ProductResponse yourPriorResponse)
             // LOGIC HERE
         }
     });
+    /* or use recommendations 
+    ps.recommendations(params, new ProductSearch.ResultListener()) {
```

Comment:
remove the searchbyid path and replace with recommendations

Reply:
changed